helper/helper.go

```diff
@@ -0,0 +1,23 @@
+package helper
+
+import (
+	"context"
+	"time"
+
+	"github.com/ONSdigital/dp-frontend-homepage-controller/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+//checks if current time is equal/after Census First Results time
+func CheckTime(cfg *config.Config) bool {
```

Comment:
It would be better to pass in a time value (of type string) instead of the config object. That way this could be used by other flags, like so:

```suggestion
func CheckTime(timeToCompare string) bool {
```

Then calling this function, you could do:

```
    ...
    enableCensusResults := helper.CheckTime(cfg.CensusFirstResults)
    ...
```

Reply:
Unit test for CheckTime function would be simpler, as there will be no need to create config object